source/process/Recruitment

```diff
@@ -0,0 +1,137 @@
+Our mission at Mattermost is to be the primary way people communicate within Enterprise and applications. 
+ Our goals are to become the leading enterprise-grade message solution, we’ll achieve this by: 
+#1 private cloud solution by deployments (12-18 months)
+#1 private cloud solution by revenue
+#1 solution by deployments (private cloud & SaaS)
+#1 solution by revenue (private cloud & SaaS)
+
+Our Values
+Values are deliberate choices defining our behavior. When facing complexity, uncertainty, or ambiguity (CUA) we determine our point of view and our actions through the lens of our values:
+
+Customer Obsession - We exist to make customers successful. In everything we do, start with customer perspective and work backwards. Earn and keep their trust.
+Ownership - Own the outcomes of your activity. Don’t drop the ball. When we see a vacuum on something important, we jump in–we never say “it’s not my job.”
```

Comment:
It should read "When we see a vacuum on something important, we jump-in, we never say “it’s not my job.”  *Note I copy pasted this from the value section on our site, if we change it here it'll have to be changed here too: https://docs.mattermost.com/process/handbook.html#values